openshift/console
# Start new export if export CR is not found

**Fixes**: 
https://issues.redhat.com/browse/ODC-6270


**Analysis / Root cause**: 
Export is not started after being deleted from another tab due to the isCreating flag which does not change to false on deletion.


**Solution Description**: 
Remove the isCreating flag, since it only has context for the current tab.


**Screen shots / Gifs for design review**: 

https://user-images.githubusercontent.com/20013884/130613023-fafd58e9-9e31-40cb-a84b-05d2d63396a5.mp4




**Unit test coverage report**: 
(Unchanged)


**Test setup:**

1. Install primer operator and create an ns and add any deployment to it
2. start an export and then click the export button again to see the modal.
3. In another window delete the export resource. 



**Browser conformance**: 


/kind bug
